types/react-draft-wysiwyg/test/basic-tests.tsx

```diff
@@ -6,8 +6,8 @@ import { Editor } from 'react-draft-wysiwyg';
 
 const Basic = () => (<div>
   <Editor
-      ref={(ref: Editor): void => {
-        console.log('hey ref', ref.focusEditor.toString());
+      ref={(ref: Editor | null): void => {
+        console.log('hey ref', ref?.focusEditor.toString());
```

Comment:
```suggestion
      ref={ ref => {
        ref; // $ExpectType Editor | null
```
this is probably everyday syntax (no explicit types)

Reply:
I didn't want to change the spirit of the test (log the stringified value). Especially without owner review. Maybe the point wasn't that `ref.focusEditor` has the `Editor` type but just that we can call `toString` on it?

Reply:
that's ok, my point is only, that regular users avoid using explicit types whenever possible (me too), so I personally promote implicit styles and simple dts assertions here, don't worry much, I know what you're working on